dojo/fixtures/initial_surveys.json

```diff
@@ -6,7 +6,7 @@
       "modified": "2015-03-30T19:31:16Z",
       "optional": false,
       "order": 1,
-      "polymorphic_ctype": 18
+      "polymorphic_ctype": 77
```

Comment:
Are the changes in this file related to the PR? If not, the changes should be reverted.